core/codec/leb128/leb128_decode_stream.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef FILECOIN_CORE_CODECS_LEB128DECODESTREAM_HPP
+#define FILECOIN_CORE_CODECS_LEB128DECODESTREAM_HPP
+
+#include <algorithm>
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include "primitives/boost_multiprecision.hpp"
+#include "primitives/decode_stream.hpp"
+
+namespace codec::leb128 {
+  template <typename T>
+  std::string toBinary(const T &t) {
+    std::string s = "";
+    int n = sizeof(T) * 8;
```

Comment:
```suggestion
    constexpr int n = sizeof(T) * 8;
```

Reply:
Converting numeric value to string was removed after refactoring